frontend/src/assets/stylesheets/components/post.sass

```diff
@@ -6,18 +11,21 @@
   padding: 32px
   position: relative
 
+  &:hover
+    +transition(background-color)
+    background-color: darken($color-black, 1%)
+
+  &__my-posts-header
+    margin: $m-lg 0
+
   &__link
-    +transition(color)
     color: $color-white
     text-decoration: none
 
-    &:hover
-      .post__text-details
-        color: $color-primary
-
   &__body
     min-height: 100px
     margin-bottom: 56px
+    color: $color-white
     p
       font-size: 20px
```

Comment:
Avoid styling by tag and consider to add variables with font-sizes like in our [boilerplate](https://github.com/Selleo/SASS-boilerplate/blob/master/app/assets/stylesheets/base/_variables.sass)